scripts/relock.js

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env node
```

Comment:
that should be fine. i was also waffling on including this file. i thought maybe it made more sense to add lockdown as a dev dependency and install it and the relock bin from there.